onnxruntime/core/session/ort_apis.h

```diff
@@ -16,6 +16,8 @@ ORT_API(void, ReleaseTypeInfo, OrtTypeInfo*);
 ORT_API(void, ReleaseTensorTypeAndShapeInfo, OrtTensorTypeAndShapeInfo*);
 ORT_API(void, ReleaseSessionOptions, OrtSessionOptions*);
 ORT_API(void, ReleaseCustomOpDomain, OrtCustomOpDomain*);
+ORT_API(void, ReleaseMapTypeInfo, OrtMapTypeInfo*);
+ORT_API(void, ReleaseSequenceTypeInfo, OrtSequenceTypeInfo*);
```

Comment:
Please talk to @RyanUnderhill  offline. We can't allow such changes.

Reply:
yep yep!